lib/Sema/CSSimplify.cpp

```diff
@@ -1989,6 +1989,17 @@ repairFailures(ConstraintSystem &cs, Type lhs, Type rhs,
     break;
   }
 
+  case ConstraintLocator::ContextualType: {
+    if (lhs->is<FunctionType>() && !rhs->is<AnyFunctionType>()) {
+      auto *fix = ContextualMismatch::create(cs, lhs, rhs,
+                                             cs.getConstraintLocator(locator));
+      conversionsOrFixes.push_back(fix);
+      break;
+    }
+
+    return;
```

Comment:
Nit: I don't think this return is necessary, just do a single `break;` at the end of this `case`.

Reply:
Fixed!